tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "@stellar/tsconfig",
   "compilerOptions": {
     "baseUrl": "src/",
-    "target": "es5",
+    "target": "es2017",
```

Comment:
This is the minimum target that supports the top-level `await` that we need for the `envVariables.ts` export.